ppr-ui/src/components/mhrTransfers/LocationChange.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="mhr-location-change mt-8">
+    <FormCard
+      :label="content.sideLabel"
+      :showErrors="false"
+      :class="{'border-error-left': false}"
+    >
+      <template #formSlot>
+        <v-select
+          id="location-change-select"
+          v-model="locationChangeType"
+          :items="roleBasedLocationChangeTypes"
+          itemTitle="title"
+          itemValue="type"
+          variant="filled"
+          label="Location Change Type"
+          color="primary"
```

Comment:
Probably won't need this. 
This sets the hint text to blue but designs in vuetify3. There is an override styling going in to fix these colours. 
Also, if this is meant to clearable, will need that prop, alongside persistentClear and the icon setting (ie mdi-close) otherwise it might be mdi-close-circle

Reply:
There is no clearable option, so this could be removed. Thanks